Aardvark/ARKEmailBugReporter.m

```diff
@@ -121,10 +121,12 @@ @implementation UIViewController (HierarchyDescription)
 
 - (void)_ARK_appendRecursiveViewControllerMappingToMapTable:(NSMapTable<UIView *, UIViewController *> *)mapTable;
 {
-    [mapTable setObject:self forKey:self.viewIfLoaded];
-    
-    for (UIViewController *childViewController in self.childViewControllers) {
-        [childViewController _ARK_appendRecursiveViewControllerMappingToMapTable:mapTable];
+    if (@available(iOS 9.0, *)) {
```

Comment:
Should we make including the view controller hierarchy a feature that's only available in iOS 9+?